### PR TITLE
Bug 1228475 - Marionette test to ensure system app should have all kinds of audio channels r=lchang

### PR DESCRIPTION
**Pull request by evanxd**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6dd42b27880cd709f64dffd4da9fb7325f5eee77

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1efcfe6d08fe6bc6a79b62b56aec993d20bbbfd9

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6ca6b246e2634110783b15dcefddddb26f863382

File: apps/system/test/marionette/audio_channel_belonged_to_system_test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/* jshint node: true*/
+/* global marionette, setup, test*/
+'use strict';
+
+var System = require('./lib/system');
+var assert = require('assert');
+
+marionette('Audio channels belonged to system app', function() {
```
Comment (luke-chang) [inline apps/system/test/marionette/audio_channel_belonged_to_system_test.js]: nit: `Audio channels that belongs to system app` or `Audio channels owned by system app`

File: apps/system/test/marionette/audio_channel_belonged_to_system_test.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  var client = marionette.client();
+  var sys;
+
+  setup(function() {
+    client.setScriptTimeout(20000);
+    sys = new System(client);
+  });
+
+  // Reenable after Bug 1235011 is fixed.
```
Comment (luke-chang) [inline apps/system/test/marionette/audio_channel_belonged_to_system_test.js]: nit: `XXX: Will re-enable it after ...`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=95c60fe5529a8dfaec8bd3bf73b8d4c353f88384

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f38c6f2c3ca2fd576d8178ac8298fa01dac8a598

**Pull request merged**